src/MatLang.jl

```diff
@@ -1,20 +1,76 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, eyeM, diagM
+
+# Native Julia functions that are just a copy:
+export zerosM, onesM, randM
+
+
+# Command Window (REPL console):
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
```

Comment:
we can provide a fallback that clears REPL console:
```julia
function clcM()
	run(`clear`)
	return nothing
end
```

Reply:
This does not work

Reply:
can you explain why this doesn't work in details? As far as I'm understanding, it would at most raise a method re-definition warning.

Reply:
It does not recognize ``clear``, it says it is undefined.

Reply:
I see, `clear` might be limited to MacOS and Linux. The point here is we need a fallback implementation. If you decide the fallback implementation to be nothing, then document it.

Reply:
It does not work inside Julia console (I use Atom inside Windows). I didn't rest on the shell or something.

Reply:
Inside Julia REPL in Windows this does not work:
```julia
 run(`cls`)
ERROR: IOError: could not spawn `cls`: no such file or directory (ENOENT)
Stacktrace:
 [1] _spawn_primitive(::String, ::Cmd, ::Array{Any,1}) at .\process.jl:400
 [2] setup_stdios(::getfield(Base, Symbol("##505#506")){Cmd}, ::Array{Any,1}) at .\process.jl:413
 [3] _spawn at .\process.jl:412 [inlined]
 [4] #run#515(::Bool, ::Function, ::Cmd) at .\process.jl:725
 [5] run(::Cmd) at .\process.jl:724
 [6] top-level scope at none:0
```

Reply:
@johnnychen94 I think it is fixed with the last commit. Could you check on your OS?